linux_service/start_moneo_services.sh

```diff
@@ -1,15 +1,25 @@
 #!/bin/bash
 PUBLISHER=$1
+MONEO_PATH=$2
 
 systemctl start moneo@node_exporter.service
 systemctl start moneo@net_exporter.service
 systemctl start moneo@nvidia_exporter.service
 
 if [[ ! -z "$PUBLISHER" ]];
 then
-    if [[ "$PUBLISHER" = true ]];
+    if [ "$PUBLISHER" = "geneva" ] && [ -d $MONEO_PATH ];
     then
+        #starts Geneva agent
+        $MONEO_PATH/src/worker/start_geneva.sh  $MONEO_PATH/src/worker/publisher/config/geneva_config.json
         sleep 5 # wait a bit for the exporters to start
         systemctl start moneo_publisher.service 
+    elif [ "$PUBLISHER" = "azure_monitor" ];
+    then
+        sleep 5 # wait a bit for the exporters to start
+        systemctl start moneo_publisher.service 
+    else
+        echo "Error: Publisher type $PUBLISHER and/or Moneo path $MONEO_PATH not recognized "
```

Comment:
Maybe specify in the error text wht the possible PUBLISHER's are, and that the MONEO_PATH is only potentially the cause of failure if the publisher is Geneva?